code/modules/mob/living/carbon/human/species_types/lizardpeople.dm

```diff
@@ -52,3 +49,6 @@
 	id = "ashlizard"
 	limbs_id = "lizard"
 	species_traits = list(MUTCOLORS,EYECOLOR,LIPS,NOBREATH,NOGUNS,DIGITIGRADE)
+
+/datum/species/lizard/ashwalker/after_equip_job(datum/job/J, mob/living/carbon/human/H)
+	H.grant_language(/datum/language/draconic)
```

Comment:
Actually ashwalkers already are given Draconic as part of their specific spawn code, because this is never called, because Ashwalkers aren't a roundstart race.

Reply:
ok give me a second